src/main/java/org/opentripplanner/updater/trip/AddedStopTime.java

```diff
@@ -19,11 +25,38 @@ final class AddedStopTime {
   @Nullable
   private final PickDrop dropOff;
 
+  @Nullable
+  public final String stopId;
+
+  @Nullable
+  private final GtfsRealtime.TripUpdate.StopTimeEvent arrival;
+
+  @Nullable
+  private final GtfsRealtime.TripUpdate.StopTimeEvent departure;
+
+  @Nullable
+  public final Integer stopSequence;
+
+  public final GtfsRealtime.TripUpdate.StopTimeUpdate.ScheduleRelationship scheduleRelationship;
+
   public static final PickDrop DEFAULT_PICK_DROP = PickDrop.SCHEDULED;
 
-  AddedStopTime(@Nullable PickDrop pickup, @Nullable PickDrop dropOff) {
+  AddedStopTime(
+    @Nullable PickDrop pickup,
+    @Nullable PickDrop dropOff,
+    @Nullable String stopId,
+    @Nullable GtfsRealtime.TripUpdate.StopTimeEvent arrival,
+    @Nullable GtfsRealtime.TripUpdate.StopTimeEvent departure,
+    @Nullable Integer stopSequence,
+    @Nullable GtfsRealtime.TripUpdate.StopTimeUpdate.ScheduleRelationship scheduleRelationship
```

Comment:
Rather than the individual fields, store the whole StopTimeUpdate and resolve the data in the getters.